permission_validators/account/remove_key_value/src/lib.rs

```diff
@@ -44,7 +44,7 @@ pub fn validate(authority: <Account as Identifiable>::Id, instruction: Instructi
     };
 
     let IdBox::AccountId(account_id) = remove_key_value.object_id()
-        .evaluate()
+        .evaluate(&Context::new())
```

Comment:
What about:

```rust
pub trait Evaluate {
    // ...
    
    fn evaluate_default<C: Context + Default>(&self) -> Result<Self::Value, Error> {
        Evaluate::evaluate(self, Context::default())
    }
}
```

Reply:
that is a good idea

Reply:
it's not possible to implement `Context::default` because it needs to get a reference to `wsv`

Reply:
Yes, but possible for wasm:: Context, I think. Because it doesn't need wsv. And that would be enough to make use out of this function

Reply:
mind you, it's part of the trait. I can't implement `Default` for `core::Context`

Reply:
You don't have to make `Default` to be a part of `Context` trait. You can just add this function with additional `Default` bound to the `Evaluate` trait

Reply:
It will be impossible to call this function for `core::Context` but will be possible for `wasm:: Context`

Reply:
let's leave it for my next PR, it'll make a difference there